index.html

```diff
@@ -840,7 +830,6 @@
                 <li>If the pointer is not in the <a>active buttons state</a>, then terminate these steps.</li>
                 <li>For the specified <code>pointerId</code>, set the <dfn>pending pointer capture target override</dfn> to the <code>Element</code> on which this method was invoked.</li>
             </ol>
```

Comment:
Do we need a line (or "note") here that pointerenter/over/leave/out events are never fired for a captured pointer?